bin/zopen-install

```diff
@@ -383,11 +441,25 @@ handlePackageInstall()
       rm -rf "${baseinstalldir}/${installdirname}"
     fi
 
-    if ! runLogProgress "pax -rf ${pax} -p p ${paxredirect} ${redirectToDevNull}" "Expanding ${pax}" "Expanded"; then
-      printWarning "Errors unpaxing, package directory state unknown."
-      printInfo "Use zopen alt to select previous version."
-      continue
+    extractPax=true
+    metadataVersion=$(jq -r '.version_scheme' "${metadataFile}" 2>/dev/null)
+    is_greater=$(echo "$metadataVersion > 0.1" | bc -l)
+
+    if [ "$is_greater" -eq 1 ] && [ "$skipverify" = "false" ]; then
+           verifySignatureOfPax
+           ret_val=$?
+           if [ $ret_val -eq 2 ]; then
+                   extractPax=false;
```

Comment:
So if the version is greater than 1, we enforce the signature verification, correct?
Should this print an error message in that case and exit?

Reply:
I noticed in verifySignatureOfPax, it can return 1 in certain error conditions. Should that case be handled here as well?